src/components/requests/Request.css

```diff
@@ -0,0 +1,7 @@
+.request-meta {
+  margin: 25px 25px 25px 0;
+  background: #fff;
+  border: 1px solid #e5e5e5;
```

Comment:
maybe out of scope for this PR but it would be nice to define the colors in a single place. :)